src/player.js

```diff
@@ -1387,6 +1389,7 @@ export default class Player extends FakeEventTarget {
     if (this._engine) {
       this._engine.enableAdaptiveBitrate();
     }
+    this._playbackAttributesState.videoHeight = undefined;
```

Comment:
why does it need to be reset ?

Reply:
to not apply default track when ABR mode is active (auto-selection according the network)

Reply:
what if we change video track, then change audio track, and then move to a new playlist item ?

Reply:
how audio track selection related to video selection?

Reply:
oh, never mind, I thought the reset was also in selectTrack